application/src/scouter/AutonomousForm.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+import SushiButton from "./SushiButton";
+import { AllSushis, AllSushisAndStorage, Sushi, ValuesToBePassed } from "./input-types/AutonomousMapInput";
+import {ShusiToBeChanged} from "./input-types/AutonomousMapInput"
+
+
+const AutonomousForm: React.FC<AllSushisAndStorage> = (props) => {
+  const sushiButtons = [<SushiButton sushies={props.allSushis} sushiToBeChanged={ShusiToBeChanged.SUSHI1} storage={props.storage} />,
+     <SushiButton sushies={props.allSushis} sushiToBeChanged={ShusiToBeChanged.SUSHI2} storage={props.storage} />,
+     <SushiButton sushies={props.allSushis} sushiToBeChanged={ShusiToBeChanged.SUSHI3} storage={props.storage}/>];
```

Comment:
```suggestion
  const sushiButtons = Object.values(props.allSushis).map((sushi) => <SushiButton sushies={props.allSushis} sushiToBeChanged={sushi}/>)
```

also change the enum `ShusiToBeChanged` to a type that is `keyof AllSushis`